packages/account/src/Components/leave-confirm/__tests__/leave-confirm.spec.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import LeaveConfirm, { TransitionBlockerWithRouter } from '../leave-confirm';
+import { Formik } from 'formik';
+import { Router } from 'react-router';
+import { createBrowserHistory } from 'history';
+import { isMobile } from '@deriv/shared';
+
+let modal_root_el;
+
+jest.mock("@deriv/shared", () => ({
+    ...jest.requireActual("@deriv/shared"),
+    isMobile: jest.fn(() => false),
+}));
+
+
+beforeAll(() => {
+    modal_root_el = document.createElement('div');
+    modal_root_el.setAttribute('id', 'modal_root');
+    document.body.appendChild(modal_root_el);
+});
+
+afterAll(() => {
+    document.body.removeChild(modal_root_el);
+});
+
+const LeaveConfirmComponent = () => {
+    const history = createBrowserHistory();
+    return (
+        <Router history={history}>
+            <Formik>
+                <LeaveConfirm />
+            </Formik>
+        </Router>
+    )
+}
+
+const withRouter = (Component) => {
+    const history = createBrowserHistory();
+    return props => {
+        return <Router history={history}>
+            <Component {...props} />
+        </Router>
+
+    }
+}
+
+const TransitionBlockerComponent = withRouter(TransitionBlockerWithRouter);
+
+describe('LeaveConfirm', () => {
+     it('should render LeaveConfirm component in desktop mode', () => {
+        jest.spyOn(React, 'useState').mockReturnValueOnce([true, () => {}])
+        render(<LeaveConfirmComponent />);
+    });
```

Comment:
There is no assertions here. So wont it get passed even if the component doesn't render?

Reply:
Added assertion here.